### PR TITLE
Update manage-phone-numbers-for-your-organization.md

### PR DESCRIPTION
**Pull request by ErsancMS**
Modifying and adding extra information that will be useful when receiving new requests here at the Service Desk

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 1]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
 
 # Manage phone numbers for your organization
 
-There are three ways to get both user and service (toll and toll-free) numbers for your organization.
-- Get numbers in the Microsoft Teams admin center.
-- Transfer or "port" your existing numbers over to us.
-- Use a request form for new numbers that aren't listed in the Microsoft Teams admin center.
+There are four ways to get user (Subscriber) and service (Toll and Toll-Free) phone numbers for your organization:
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:32]:
```suggestion
There are four ways to get user (Subscriber) and service (toll and toll-free) phone numbers for your organization:
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 2]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
...
+- [Getting **new user** phone numbers in the Microsoft Teams Admin Center](https://docs.microsoft.com/en-us/microsoftteams/getting-phone-numbers-for-your-users#get-new-phone-numbers-for-your-users).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:33]:
```suggestion
- [Getting **new user** phone numbers in the Microsoft Teams Admin Center](https://docs.microsoft.com/microsoftteams/getting-phone-numbers-for-your-users#get-new-phone-numbers-for-your-users).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 3]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
...
+- [Getting **new service** phone numbers in the Microsoft Teams Admin Center](https://docs.microsoft.com/en-us/microsoftteams/getting-service-phone-numbers#get-new-service-numbers).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:34]:
```suggestion
- [Getting **new service** phone numbers in the Microsoft Teams Admin Center](https://docs.microsoft.com/microsoftteams/getting-service-phone-numbers#get-new-service-numbers).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 4]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
...
+- [**Transfer or _Port_** your existing phone numbers with us](https://docs.microsoft.com/en-us/microsoftteams/phone-number-calling-plans/transfer-phone-numbers-to-teams#create-a-port-order-and-transfer-your-phone-numbers-to-teams).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:35]:
```suggestion
- [**Transfer or _Port_** your existing phone numbers with us](https://docs.microsoft.com/microsoftteams/phone-number-calling-plans/transfer-phone-numbers-to-teams#create-a-port-order-and-transfer-your-phone-numbers-to-teams).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 5]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
...
+- Use a request form for new phone numbers that aren't listed in the Microsoft Teams admin center.
 
 > [!IMPORTANT]
-> In Belgium, France, Germany, Ireland, the Netherlands, and Spain, you can't get phone new phone numbers using the Microsoft Teams admin center. You have to download and fill out a form. For new phone numbers in Australia, see [Telstra Voice Services](https://aka.ms/TelstraVoicePlan).
+> In Belgium, France, Germany, Ireland, the Netherlands, and Spain, you can't get phone new phone numbers using the Microsoft Teams Admin Center. You have to download and fill out a form. For new phone numbers in Australia, see: [**Telstra Voice Services**](https://aka.ms/TelstraVoicePlan).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:39]:
```suggestion
> In Belgium, France, Germany, Ireland, the Netherlands, and Spain, you can't get new phone numbers by using the Microsoft Teams admin center. You have to download and fill out a form. For new phone numbers in Australia, see [Telstra Voice Services](https://aka.ms/TelstraVoicePlan).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 6]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
...
+- [**Transfer or _Port_** your existing phone numbers with us](https://docs.microsoft.com/microsoftteams/phone-number-calling-plans/transfer-phone-numbers-to-teams#create-a-port-order-and-transfer-your-phone-numbers-to-teams).
+- Use a request form for new phone numbers that aren't listed in the Microsoft Teams admin center.
 
 > [!IMPORTANT]
-> In Belgium, France, Germany, Ireland, the Netherlands, and Spain, you can't get phone new phone numbers using the Microsoft Teams admin center. You have to download and fill out a form. For new phone numbers in Australia, see [Telstra Voice Services](https://aka.ms/TelstraVoicePlan).
+> In Belgium, France, Germany, Ireland, the Netherlands, and Spain, you can't get phone new phone numbers using the Microsoft Teams Admin Center. You have to download and fill out a form. For new phone numbers in Australia, see: [**Telstra Voice Services**](https://aka.ms/TelstraVoicePlan).
 
 ***
 If you know what kind of forms you are looking for use the drop-down to select the country/region where you're getting numbers.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:42]:
```suggestion
If you know what kind of forms you are looking for, use the drop-down to select the country/region where you're getting numbers.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 7]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+> - [U.S. & Puerto Rico](phone-number-management-for-the-u-s.md)
 
 ***
-## Here's a bit more information about these number request forms
+## Here's a bit more information about these Number Request Forms
 
 ### Request forms for new phone numbers
 
-Each country or region has different instructions, different types of numbers (geographic/non-geographic and service (toll/toll-free) and rules/regulations for getting phone numbers so they can be used in Teams.
+Each country or region has different instructions, different types of phone numbers (Geographic/Non-Geographic and service (Toll/Toll-Free)) and rules/regulations for getting phone numbers so they can be used in Microsoft Teams.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:63]:
```suggestion
Each country or region has different instructions, different types of phone numbers (geographic/non-geographic and service (toll/toll-free)) and rules/regulations for getting phone numbers so they can be used in Microsoft Teams.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 8]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
 
-Sometimes (depending on your country/region) you won't be able to get new user (subscriber) or service (toll or toll-free) numbers using the Microsoft Teams admin center, or you might need specific phone numbers or area codes.
+Sometimes (depending on your country or region) you won't be able to get new **user** (Subscriber) or **service** (Toll or Toll-Free) phone numbers using the Microsoft Teams Admin Center or you might need _specific_ phone numbers (vanity requests) or _specific_ area codes.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:65]:
```suggestion
Sometimes (depending on your country or region) you won't be able to get new **user** (Subscriber) or **service** (toll or toll-free) phone numbers using the Microsoft Teams admin center or you might need _specific_ phone numbers (vanity requests) or _specific_ area codes.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 9]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
 
-If so, you'll need to download the correct PDF form (per country/region and the type of numbers) and send it back to us. Our service desk will take it from there and will let you know about the progress or your order.
+If so, you will need to download the correct PDF LOA form (per country or region and type of phone numbers) and send it back to us. Our Service Desk will take it from there and start processing your request.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:67]:
```suggestion
If so, you will need to download the correct PDF LOA form (per country or region and type of phone numbers) and send it back to us. Our service desk will take it from there and start processing your request.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 10]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
 
-### Letters of Authorization (LOAs) for transferring numbers
+### Letters of Authorization (LOAs) to transfer/port existing phone numbers from another service provider to Microsoft Teams
 
-If you need to transfer phone numbers, you'll to download a **Letter of Authorization (LOA)**, which gives us permission to request, on your behalf, transferring the existing number(s) from a different service provider in case of a dispute. Use an LOA if you have:
-- User numbers that you already have from another carrier.
-- Service (toll) numbers for audio conferencing bridges, auto attendants or call queues.
-- Service (toll-free) phone numbers.
-- You have more than 999 user phone numbers that you need to get but can't get in the Microsoft Teams admin center using the use the Local Number Porting wizard.
+If you need to transfer/port your existing phone numbers, you will need to: download, fill and send us a **Letter of Authorization (LOA)** (which gives us permission to request and transfer -on your behalf- those existing numbers from a different service provider into Microsoft Teams).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:71]:
```suggestion
If you need to transfer/port your existing phone numbers, you will need to download, fill and send us a **Letter of Authorization (LOA)** (which gives us permission to request and transfer, on your behalf, those existing numbers from a different service provider into Microsoft Teams).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 11]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
 
-## If you still need help and aren't sure what to do....
+Please send us an **LOA** if you currently have with another carrier:
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:73]:
```suggestion
Please send us an **LOA** if you currently have the following with another carrier:
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 12]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+- User (Subscriber) numbers.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:74]:
```suggestion
- User (subscriber) numbers.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 13]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+- Service (Toll) numbers for audio conferencing bridges, auto attendants or call queues.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:75]:
```suggestion
- Service (toll) numbers for audio conferencing bridges, auto attendants, or call queues.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 14]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+- Toll-Free phone numbers.
+- More than 999 user phone numbers that you need to transfer/port but can't get in the Microsoft Teams admin center using the use the Local Number Porting wizard.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:77]:
```suggestion
- More than 999 user phone numbers that you need to transfer/port but can't get in the Microsoft Teams admin center using the local number porting wizard.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 15]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
 
-If you still need help, let us know how we can help you. In your Help request, include details such as your organization ID, domains, what types of numbers you need help with, how many numbers you want, authorizing person on the account and other important details about your issue.
-- For help with phone numbers inside the U.S., [send your request](mailto:ptn@microsoft.com).
-- For help with phone numbers in Europe, [send your request](mailto:ptneu@microsoft.com).
-- For help with phone number in Asia, [send your request](mailto:ptnapac@microsoft.com).
+Once you have your **filled and signed** LOA, please [**contact our PSTN Service Desk**](https://docs.microsoft.com/en-us/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk) so we can assist you.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:79]:
```suggestion
Once you have your **filled and signed** LOA, please [contact our PSTN Service Desk](https://docs.microsoft.com/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk) so we can assist you.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 16]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
 
-#### Additional resources
+## Still need assistance?
 
-- To see if Calling Plans are available in your country or region, see [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md)
-- To help you pick the type of phone number you need, read [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
-- To learn about getting user phone numbers in the admin center, read [Getting phone numbers for your users](/microsoftteams/getting-phone-numbers-for-your-users).
-- To get help setting up phone numbers for your organization, [contact the PSTN service desk](contact-pstn-service-desk.md).
-  
+[**Please contact our PSTN Service Desk**](https://docs.microsoft.com/en-us/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk) and we will assist you as soon as possible.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:83]:
```suggestion
[**Please contact our PSTN Service Desk**](https://docs.microsoft.com/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk) and we will assist you as soon as possible.
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 17]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+
+## Additional resources
+
+- To see _if_ and which **Calling Plans** are available in your country or region, read: [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md)
+- To assist you on choosing the correct type of phone numbers, read: [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:88]:
```suggestion
- To assist you in choosing the correct type of phone numbers, read [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 18]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+- To get assistance setting up phone numbers for your organization, read: [Support Contact for Business Products - Admin Help
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:89]:
```suggestion
- To get assistance with setting up phone numbers for your organization, read [Support Contact for Business Products - Admin Help](https://docs.microsoft.com/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 19]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:90]:
```suggestion
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 20]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
 
 # Manage phone numbers for your organization
 
-There are three ways to get both user and service (toll and toll-free) numbers for your organization.
-- Get numbers in the Microsoft Teams admin center.
-- Transfer or "port" your existing numbers over to us.
-- Use a request form for new numbers that aren't listed in the Microsoft Teams admin center.
+There are four ways to get user (Subscriber) and service (Toll and Toll-Free) phone numbers for your organization:
+- [Getting **new user** phone numbers in the Microsoft Teams Admin Center](https://docs.microsoft.com/microsoftteams/getting-phone-numbers-for-your-users#get-new-phone-numbers-for-your-users).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:33]:
```suggestion
- [Getting new user phone numbers in the Microsoft Teams admin center](https://docs.microsoft.com/microsoftteams/getting-phone-numbers-for-your-users#get-new-phone-numbers-for-your-users).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 21]
```diff
@@ -29,18 +29,19 @@ description: Learn how to get and manage phone numbers for Microsoft Teams for y
...
+- [Getting **new service** phone numbers in the Microsoft Teams Admin Center](https://docs.microsoft.com/en-us/microsoftteams/getting-service-phone-numbers#get-new-service-numbers).
+- [**Transfer or _Port_** your existing phone numbers with us](https://docs.microsoft.com/microsoftteams/phone-number-calling-plans/transfer-phone-numbers-to-teams#create-a-port-order-and-transfer-your-phone-numbers-to-teams).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:35]:
```suggestion
- [Transfer or _Port_ your existing phone numbers with us](https://docs.microsoft.com/microsoftteams/phone-number-calling-plans/transfer-phone-numbers-to-teams#create-a-port-order-and-transfer-your-phone-numbers-to-teams).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 22]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
 > - [Netherlands](phone-number-management-for-the-netherlands.md)
 > - [Spain](phone-number-management-for-spain.md)
 > - [U.K.](phone-number-management-for-the-u-k.md)
-> - [U.S. (Puerto Rico)](phone-number-management-for-the-u-s.md)
+> - [U.S. & Puerto Rico](phone-number-management-for-the-u-s.md)
 
 ***
-## Here's a bit more information about these number request forms
+## Here's a bit more information about these Number Request Forms
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:59]:
```suggestion
## Here's a bit more information about these number request forms
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 23]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
-- To help you pick the type of phone number you need, read [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
-- To learn about getting user phone numbers in the admin center, read [Getting phone numbers for your users](/microsoftteams/getting-phone-numbers-for-your-users).
-- To get help setting up phone numbers for your organization, [contact the PSTN service desk](contact-pstn-service-desk.md).
-  
+[**Please contact our PSTN Service Desk**](https://docs.microsoft.com/en-us/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk) and we will assist you as soon as possible.
+
+## Additional resources
+
+- To see _if_ and which **Calling Plans** are available in your country or region, read: [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:87]:
```suggestion
- To see _if_ and which calling plans are available in your country or region, read [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md)
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md (hunk 19 above)
Comment (ErsancMS) [reply Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:90]: @JohanFreelancer9  please put in this:
- To get assistance setting up phone numbers for your organization, read: [Support Contact for Business Products - Admin Help](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md (hunk 18 above)
Comment (ErsancMS) [reply Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:89]: @JohanFreelancer9 that is not the correct link for Support Contact for Business Product. please put in the following:
- To get assistance with setting up phone numbers for your organization, read [Support Contact for Business Products - Admin Help](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).

Comment (JohanFreelancer9) [reply Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:89]: Done.

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md (hunk 19 above)
Comment (ErsancMS) [reply Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:90]: @JohanFreelancer9 how should we go about making the right change on this one?
Line that starts with: - To get assistance setting up phone numbers for your organization
Should be:
- To get assistance setting up phone numbers for your organization, read: [Support Contact for Business Products - Admin Help](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 24]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
-- To get help setting up phone numbers for your organization, [contact the PSTN service desk](contact-pstn-service-desk.md).
-  
+[Please contact our PSTN Service Desk](https://docs.microsoft.com/microsoftteams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk) and we will assist you as soon as possible.
+
+## Additional resources
+
+- To see _if_ and which calling plans are available in your country or region, read [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md)
+- To assist you in choosing the correct type of phone numbers, read [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
+- To get assistance with setting up phone numbers for your organization, read [Contact support for business products - Admin Help](https://docs.microsoft.com/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:89]:
```suggestion
- To get assistance with setting up phone numbers for your organization, read: [Contact support for business products - Admin Help](https://docs.microsoft.com/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md [hunk 25]
```diff
@@ -52,44 +53,44 @@ If you know what kind of forms you are looking for use the drop-down to select t
...
+](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:90]:
```suggestion
```

File: Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md (hunk 19 above)
Comment (JohanFreelancer9) [reply Teams/manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md:90]: @ErsancMS If you merge my last two commits, that should do it. Thanks.

**Pull request merged**